test/integration/targets/nxos_logging/tasks/main.yaml

```diff
@@ -1,3 +1,14 @@
 ---
-- { include: cli.yaml, tags: ['cli'] }
-- { include: nxapi.yaml, tags: ['nxapi'] }
+# Use block to ensure that both cli and nxapi tests
+# will run even if there are failures or errors.
+- block:
+  - { include: cli.yaml, tags: ['cli'] }
+  rescue:
```

Comment:
Fix coming for this.  I plan to move both cli and nxapi under the block and get rid of the rescue.